tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_health_counters.py

```diff
@@ -54,12 +85,22 @@ def get_metrics(self):
                     {}
                 ).iteritems():
                     for brick in sub_volume_bricks:
-                        brick_ip = socket.gethostbyname(brick.get('hostname'))
+                        brick_hostname = \
+                            tendrl_glusterfs_utils.find_brick_host(
+                                self.etcd_client, brick.get('hostname')
+                            )
+                        if brick_hostname:
+                            brick_ip = socket.gethostbyname(brick_hostname)
+                        else:
+                            brick_ip = socket.gethostbyname(
+                                brick.get('hostname')
+                            )
+                            brick_hostname = brick.get('hostname')
```

Comment:
This code snippet seems similar to the one I have commented below, can you please check if both are the same and make a common util or something if they are doing the same thing